### PR TITLE
ChangeWave 16.8 Becomes Default Behavior

### PR DESCRIPTION
**Pull request by BenVillalobos**
### Context
Features under ChangeWave 16.8 are due for becoming normal functionality.

### Changes Made
Removed Wave_16.8 and all checks/tests that had to do with changewave16.8.

### Notes

### CODE REVIEW

**Review by rainersigwald — CHANGES_REQUESTED** [review]
Can you update the doc, and also update the doc with the lifespan of a changewave? I had a vague recollection that these would last another two point releases.

**Comment by BenVillalobos** [comment]
@rainersigwald Had to remind myself as well. Turns out we decided to make waves standard after two waves.

File: documentation/wiki/ChangeWaves.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ Opt-out is a better approach for us because we'd likely get limited feedback whe
 ## How do they work?
 The opt-out comes in the form of setting the environment variable `MSBuildDisableFeaturesFromVersion` to the Change Wave (or version) that contains the feature you want **disabled**. This version happens to be the version of MSBuild that the features were developed for. See the mapping of change waves to features below.
 
+## When do they become permanent?
+A wave of features is set to "rotate out" (ie. become standard functionality) two bands after its release. For example, wave 16.8 stayed opt-out through wave 16.10, becoming standard functionalty when wave 17.0 is introduced.
```
Comment (rainersigwald) [inline documentation/wiki/ChangeWaves.md:11]:
```suggestion
A wave of features is set to "rotate out" (i.e. become standard functionality) two bands after its release. For example, wave 16.8 stayed opt-out through wave 16.10, becoming standard functionalty when wave 17.0 is introduced.
```

**Review by rainersigwald — APPROVED** [review]

**Review by Forgind — APPROVED** [review]
💗 deleting code 🙂

**Pull request merged**